src/data/finset.lean

```diff
@@ -2321,6 +2404,17 @@ begin
     apply lt_of_lt_of_le H4 H6 }
 end
 
+/--
+If there's more than 1 element, the min' is different to the max'. An alternate version of
+`min'_lt_max'` which is sometimes more convenient.
+-/
+lemma min_ne_max_of_card (h₂ : 1 < card s) : s.min' H ≠ s.max' H :=
+begin
+  intro, apply not_le_of_lt h₂ (le_of_eq _), rw card_eq_one,
```

Comment:
In a proof which is not a one-liner, there should be one command per line (and if additional goals do show up they should be proved in enclosing braces).